src/Traits/ReviewPermission.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace SilverStripe\ContentReview\Traits;
+
+use SilverStripe\CMS\Model\SiteTree;
+use SilverStripe\ORM\DataObject;
+use SilverStripe\Security\Member;
+use SilverStripe\Security\Security;
+
+trait ReviewPermission
```

Comment:
```suggestion
trait PermissionChecker
```
Also remember to update the filename and references to this trait

Reply:
done